content/installation/net/install/Installer-CommandLine-Options.md

```diff
@@ -0,0 +1,38 @@
+<!--
+title: "Contrast .NET Agent Installer Command Line Options"
+description: "Contrast .NET agent installer command line options"
+tags: "installation agent .NET windows silent unattended installer setup"
+-->
+
+## .NET Agent Installer Command Line Options
+
+The .NET agent installer supports additional options when invoked from the command line.
+
+### Silent Installation
+
+The .NET agent installer supports standard silent installation options. These options tell the installer to run silently and unattended, which means that the installer don't require your interaction and don't present the installer's UI.
+
+- Install: `ContrastSetup.exe -s -norestart`
+
+### Uninstalling and Repairing
+
+The .NET agent installation can be uninstall or repaired using standard Windows features (e.g. the Programs and Features Control Panel, Powershell, etc.). However, it may be desirable to use the .NET agent installer to perform these actions instead.
+
+**Attended:**
+- Uninstall: `ContrastSetup.exe -uninstall`
```

Comment:
Lets add one for "Install" too.
Do we need repair?  I don't know if that's widely used or people know what it does.

Reply:
Install, you mean the Silent Installation section :D 

I like keeping repair since it's stating we do support standard MSI flags. We have to support it anyway because we can't prevent users from using that feature, so we might as well advertise support.

Reply:
I think it would read better with all 3 options in the same table and using the same terminology.  The "silent" installation is the same thing as unattended here, and there is no command for "attended".

Reply:
The lack of a silent flag is attended. In a lot of installers, most of Microsoft's installers, unattended != silent. I just want to make it plain that you can use some of the flags with using the UI to install e.g. the service startup flag can be used with the UI.

Reply:
@Silvenga Is this resolved?

Reply:
Yes, I merged the install/uninstall sections.